scripts/seed-probely-db.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+BACKUP=${1:-latest.dump}
+python -m scripts.resetdb --skip-db-creation
+pg_restore --verbose --clean --no-acl --no-owner -U arlo -d $DATABASE_URL $BACKUP
```

Comment:
I think we could replace the contents of restore-db-backup.sh with this script if we just add this line in:
```bash
DATABASE_URL=${DATABASE_URL:-postgresql://arlo:arlo@localhost:5432/arlo}
```

Then we don't need two separate scripts!

Reply:
`dropdb` and `createdb` are Postgres tools. `resetdb` looks like it's calling similar functions but using Python libraries. There may be some differences, but if it works I wouldn't worry about it too much.